index.html

```diff
@@ -28,31 +28,32 @@
             </div>
         </nav>
     </header>
-    <main class="d-flex flex-column align-items-center py-4">
-        <h1 class="align-self-start mb-4 w-50 align-self-center">Home <span class="text-muted fs-3">| Wall of Quacks</span></h1>
-        <form id="write-message" class="card mb-3 w-50 d-flex flex-column p-3">
-            <div class="d-flex">
-                <div class="col-md-1 me-2">
+    <main class="py-4">
+      <div class="d-flex flex-column align-items-center">
+        <h1 class="col-md-6 col-sm-8 col-9 align-self-start mb-4 align-self-center">Home <span class="text-muted fs-3">| Wall of Quacks</span></h1>
+        <form id="write-message" class="card mb-3 col-md-6 col-sm-8 col-11 d-flex flex-column p-3">
+            <div class="d-flex m-1 gap-1">
```

Comment:
```suggestion
            <div class="row m-1 gap-1">
```
This is the same issue. Rows and columns are always used in pairs:

```html
<div class="row">
  <div class="col-x-y"></div>
  <div class="not-a-col"></div> <-- invalid
</div>

<div class="not-a-row">
  <div class="col-x-y"></div> <-- invalid
</div>
```